client/resources/js/main.js

```diff
@@ -1,52 +1,54 @@
 (() => {
     "use strict";
     document.addEventListener('DOMContentLoaded',initCoordinates)
-    
+
         // check if browser supports the geolocation api
         function initCoordinates(){
             if(navigator.geolocation) {
 
                navigator.geolocation.getCurrentPosition(initMap);			// if geolocation supported, call function
-           
+
             } else {
                 $("#location").val('Your browser doesn\'t support the geolocation api.');
             }
         }
-                
+
         function initMap(position) {
             let latitude		= position.coords.latitude;				// set latitude variable
             let longitude		= position.coords.longitude;
             console.log(latitude +'\n' + longitude);			// set longitude variable
-            
+
             let mapcanvas		= document.createElement('div');		// create div to hold map
             mapcanvas.id = 'map';										// give this div an id of 'map'
-            mapcanvas.style.height = '900px';							// set map height
+            mapcanvas.style.height = '100%';							// set map height
```

Comment:
How about we change to 100vh to reach a full-screen-width?
```suggestion
            mapcanvas.style.height = '100vh';							// set map height
```